js/scripts.js

```diff
@@ -0,0 +1,14 @@
+ getTriangleArea = function getTriangleArea(a, h) {
+
+	if ( (a <= 0) && (h <= 0) ) {
+	console.log('Nieprawidłowe dane')
+}
+	return a*h/2 ;
+}
+
+// zmienne pola trójkąta
+
+console.log( getTriangleArea(10,6) )
+console.log( getTriangleArea(5,6) )
+console.log( getTriangleArea(2,6) )
```

Comment:
przetestuj wartości ujemne, według mnie jak podasz w *jednym* z argumentów wartość ujemną to nie dostaniesz wiadomości o nieprawidłowych danych (a powinieneś) - pokombinuj dlaczego. Przy dwóch wartościach ujemnych podanych będzie już pewnie OK.